configs/yolo/yolov3_d53_mstrain-608_273e_coco.py

```diff
@@ -0,0 +1,117 @@
+_base_ = [
+    '../_base_/default_runtime.py',
+]
+# model settings
+model = dict(
+    type='YOLOV3',
+    pretrained='open-mmlab://darknet53',
+    backbone=dict(
+        type='Darknet',
+        depth=53,
+        out_indices=(3, 4, 5),
+    ),
+    neck=dict(
+        type='YOLOV3Neck',
+        num_scales=3,
+        in_channels=[1024, 512, 256],
+        out_channels=[512, 256, 128],
```

Comment:
comma here is unnecessary, please clean these unnecessary commas in all the configs.